### ISSUE TITLE
Error rustdesk pro S6 - 1.1.14-4

### ISSUE THREAD

**Issue by Sebshino** [rustdesk/rustdesk-server-pro#197]
Hello,

I have the impression that there is a problem with the container of the latest version of rustdesk pro S6 - 1.1.14-4.

I tried to install it on Synology 3 and I got the error message:

/usr/bin/hbbr: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory.

Have you encountered the problem?

edit: no problem with the other containers version....

Sincerely,
Seb

**Comment by rustdesk** [comment]
Yes, this is introduced in 1.1.14-4, I did test the docker, but not S6, because I thought no users use it. I will fix it.

**Comment by Sebshino** [comment]
Do you have any idea on the repair time?

for the moment I have switched to an opensource version in order to be able to work....

**Comment by rustdesk** [comment]
For hbbr, the Pro and the OSS is the same.

**Comment by Sebshino** [comment]
<html><body>

2024/02/06 08:55:07 | stdout | /usr/bin/hbbr: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory
-- | -- | --
2024/02/06 08:55:06 | stdout | /usr/bin/hbbs: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory
2024/02/06 08:55:06 | stdout | /usr/bin/hbbr: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory
2024/02/06 08:55:05 | stdout | /usr/bin/hbbs: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory
2024/02/06 08:55:05 | stdout | /usr/bin/hbbr: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory
2024/02/06 08:55:04 | stdout | /usr/bin/hbbs: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory
2024/02/06 08:55:04 | stdout | /usr/bin/hbbr: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory
2024/02/06 08:55:03 | stdout | /usr/bin/hbbs: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory


</body>
</html>

the error is also on the hbbs

**Comment by rustdesk** [comment]
Could you use the standard Pro docker instead?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server-pro/releases/tag/1.1.14-6

**Comment by Sebshino** [comment]
thank you for your responsiveness, I have updated the different sites, however I have the impression that the client connection is very random..... as if the client had difficulty connecting to the server while it is well marked "ready".

same symptom with the classic pro version

**Comment by Sebshino** [comment]
I don't know why but it takes a lot of time for customers to upgrade to the pro version with the version and most of the time it marks non-existent ID....

I didn't have the problem before.....

**Comment by rustdesk** [comment]
https://hub.docker.com/r/rustdesk/rustdesk-server-pro-s6/tags?page=2

**Comment by rustdesk** [comment]
new s6 uses ubuntu as base， old uses busybox， maybe ubuntu is heavy for your synogy.

**Comment by rustdesk** [comment]
@dinger1986 do you have this issue if upgrading to newest Pro.

**Comment by dinger1986** [comment]
I dont use docker, I did have huge issues with docker on synology when I did testing before and couldnt get it to work 100% something on synology stopped it as I tried other docker systems as well and they were also blocked

**Comment by Sebshino** [comment]
I had no problem with docker before version 1.1.14-1

**Comment by Sebshino** [comment]
I test the reinstallation on a later version, however I use a syno 920+ (Intel Celeron 2.7Ghz + 8GB RAM + 512GB NVME Cache)

**Comment by Sebshino** [comment]
everything is functional again with an old version
